src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -41,6 +39,7 @@
     protected static final String MESSAGE_ERROR_NO_ITEM_NUM = "Please input the item number!";
     protected static final String MESSAGE_ERROR_INVALID_ITEM_NUM = "Please input the item number as a number! E.g 1";
     protected static final String MESSAGE_ERROR_NOT_A_NUMBER = "Please input a number!";
+    public static final int PARAMS_ALL_INDICES = 0;
```

Comment:
Its good to remove magic numbers. Good job!

Reply:
Agreed! Nice! Though we should standardise it and use it in `DeleteExerciseCommand` as well 👍